api/src/controllers/publications.js

```diff
@@ -4,10 +4,12 @@
  */
 const mongoose = require('mongoose');
 const logger = require('winston');
-const { firefox } = require('playwright');
+const axios = require('axios');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
```

Comment:
A dumb question regarding Git, aren't the changes in this file (and other publication files) already in main? Why is it still showing in Files Changed?

Reply:
I just noticed this 🤣 . Was it because I merged main into this branch?

Reply:
Update - Merging normally seems to do the trick, the more you know I guess..

Reply:
Yeah, I've done a squash merge from main into my branch before and cause it checks the diffs via the commit hash, everything is in a new commit hash after you squash and thinks its new. :/ Alas.